packages/ui/src/components/ListV2/Row.tsx

```diff
@@ -0,0 +1,206 @@
+import styled from '@emotion/styled'
+import type { ForwardedRef, ReactNode } from 'react'
+import { forwardRef, useEffect } from 'react'
+import { SENTIMENTS } from '../../theme'
+import { Checkbox } from '../Checkbox'
+import { Tooltip } from '../Tooltip'
+import { Cell } from './Cell'
+import { useListContext } from './ListContext'
+
+const ExpandableWrapper = styled.div`
+  grid-column: 1 / -1;
+  grid-column-start: 1;
+  border-top: 1px solid ${({ theme }) => theme.colors.neutral.borderWeak};
+  margin: 0 -${({ theme }) => theme.space['2']};
+  padding: ${({ theme }) => theme.space['2']};
+`
+
+export const StyledRow = styled.div`
+  /* List itself also apply style about common templating between HeaderRow and other Rows */
+
+  position: relative;
+  border: 1px solid ${({ theme }) => theme.colors.neutral.borderWeak};
+  border-radius: ${({ theme }) => theme.radii.default};
+  transition: box-shadow 200ms ease, border-color 200ms ease;
+  box-shadow: none;
+  background-color: ${({ theme }) => theme.colors.neutral.background};
+  cursor: auto;
```

Comment:
It's already the initial value, is there any use for it?

![Screenshot 2023-02-22 at 11 30 29](https://user-images.githubusercontent.com/15812968/220594342-607b792b-12da-4a37-ae9b-dc61baddd6a2.png)

Reply:
was similar in the current ListV2 (ListRow), i change